md/custom-widgets.md

```diff
@@ -81,6 +81,18 @@ Template:
 </div>
 ```
 
+**Note:**
+If you want use _carousel_ and _slide_ directives in same page as bonita widgets who import `ui-bootstrap-tpls-0.13.0.min.js`, 
```

Comment:
I would say
in **the** same page as widgets **that** import `ui-bootstrap-tpls-0.13.0.min.js` **as asset.**